psycop/common/feature_generation/sequences/patient_loaders.py

```diff
@@ -13,13 +13,23 @@
     EventDataFramesToPatientSlices,
 )
 
+from ...sequence_models.registry import Registry
+
+T = TypeVar("T")
+
 
 class EventDfLoader(ABC):
     @abstractmethod
     def load_events(self) -> pl.LazyFrame:
         ...
 
 
+@Registry.utilities.register("list_creator")
+def list_creator(*args: T) -> list[T]:
```

Comment:
So here we just say that the type should be the same? (I am not sure we want to say that)

Reply:
If the input is `list_creator(1, "2")`, the type of `*args` is `int | str`. I think the return type should match that :-)

Reply:
(Have refactored in dependent PR to move it to `registry.py`, deleting here)